applications/asset_tracker_v2/tests/cloud_codec/cloud_codec_nrf/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(cloud_codec_nrf_test)
+
+set(ASSET_TRACKER_V2_DIR ../..)
+set(NRF_SDK_DIR ${ASSET_TRACKER_V2_DIR}/../../../)
```

Comment:
You can remove this and use the `NRF_DIR` variable which is already set for us

Reply:
NRF_DIR is not the base directory I'm looking for. `nrfxlib` is in the same directory as `nrf`.